src/dataflow/connect.rs

```diff
@@ -64,6 +66,8 @@ where
     write_stream
 }
 
+/// Adds a parallel sink operator, which receives data on input read streams and directly
+/// interacts with external systems.
```

Comment:
Can we refer to the actual types of the operators in these documentations so developers can click and see the interface?